metricbeat/helper/elastic/elastic_test.go

```diff
@@ -140,3 +142,81 @@ func TestFixTimestampField(t *testing.T) {
 		})
 	}
 }
+
+func TestReconfigureXPackEnabledMetricSets(t *testing.T) {
```

Comment:
```suggestion
func TestConfigureModule(t *testing.T) {
```
If not it looks like we are testing `Reconfigure`, but there is already a test for that in `metricbeat/mb/mb_test.go`.

Reply:
Ah yes, sorry, this is a leftover from previous iterations of the implementation. Good catch, thanks, will fix!

Reply:
Fixed in 81e1d967a.